src/frontend-samples/global-display-sample/GlobalDisplayUI.tsx

```diff
@@ -0,0 +1,188 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { BackgroundMapType } from "@bentley/imodeljs-common";
+import { IModelApp, IModelConnection, ScreenViewport, SpatialViewState } from "@bentley/imodeljs-frontend";
+import "common/samples-common.scss";
+import { ControlPane } from "common/ControlPane/ControlPane";
+import { SandboxViewport } from "common/SandboxViewport/SandboxViewport";
+import * as React from "react";
+import { Button, Input, Toggle } from "@bentley/ui-core";
+import GlobalDisplayApp from "./GlobalDisplayApp";
+
+interface GlobalDisplayUIState {
+  viewport?: ScreenViewport;
+  /** Place name to which to travel. */
+  destination: string;
+  /** True for 3d terrain, false for a flat map. */
+  terrain: boolean;
+  /** Display map labels with the map imagery. */
+  mapLabels: boolean;
+  /** Display 3d building meshes from Open Street Map Buildings. */
+  buildings: boolean;
+  /** If buildings are displayed, also display their edges. */
+  buildingEdges: boolean;
+}
+
+interface GlobalDisplayUIProps {
+  iModelSelector: React.ReactNode;
+  iModelName: string;
+}
+
+export default class GlobalDisplayUI extends React.Component<GlobalDisplayUIProps, GlobalDisplayUIState> {
+  public state: GlobalDisplayUIState = {
+    destination: "",
+    terrain: false,
+    buildings: false,
+    buildingEdges: false,
+    mapLabels: true,
+  };
+
+  private readonly onIModelReady = () => {
+    IModelApp.viewManager.onViewOpen.addOnce((viewport: ScreenViewport) => {
+      // The grid just gets in the way - turn it off.
+      viewport.view.viewFlags.grid = false;
```

Comment:
@Ellord207, were you working on a generalized solution to the grid display?

Reply:
Yes.  Turns out it should already be turned off, but there's a bug in the sample showcase keeping that from happening.   I was a planning adding more to that PR before merging it.

If you want to leave this here, I can remove it with my changes.  If you do, please clone the flags then reassign the flags after the changes.  Not doing so created the bug in the first place.

Reply:
@pmconne, I have completed my PR fixing the grid in the showcase.  It should be disabled by default for all samples.

https://github.com/imodeljs/frontend-sample-showcase/pull/177